api/fittings/lisk_cache.js

```diff
@@ -13,9 +13,26 @@
  */
 'use strict';
 
-var modules = require('../../helpers/swagger_module_registry');
 var debug = require('debug')('swagger:lisk:cache');
+var modules = require('../../helpers/swagger_module_registry');
 
+/**
+ * Description of the module.
+ * @module
+ * @requires debug
+ * @requires helpers/swagger_module_registry.getCache
+ * @requires helpers/swagger_module_registry.getLogger
+ * @todo: add description of the module
+ */
+
+/**
+ * Description of the function.
+ * @func create
+ * @param {Object} fittingDef - Description of the param.
+ * @param {Object} bagpipes - Description of the param.
+ * @returns {function} lisk_cache
+ * @todo: add description of the function and its parameters
+ */
```

Comment:
In `api/controllers` we have a `@module` with `@param`s, whereas here we have two definitions. Is it important to distinguish the module from the function here?

Reply:
It is not important, in fact I will restructure it a little bit according to your feedback, see next commits :)